src/Generator/PropertyGenerator.php

```diff
@@ -4,13 +4,17 @@
 
 use Laminas\Code\Reflection\PropertyReflection;
 
+use function array_reduce;
+use function method_exists;
+use function PHPUnit\Framework\assertIsBool;
```

Comment:
This symbol comes from the test suite: we can't use it in sources

Reply:
:sweat_smile: 
Removed and replaced by condition and exception.